service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java

```diff
@@ -40,6 +40,7 @@
     private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
     private final SendDocumentService sendDocumentService;
 
+    // REFACTOR: 04/06/2021 change private methods into their own event listeners
```

Comment:
can you create a tech debt ticket and remove the comment?

Reply:
made https://tools.hmcts.net/jira/browse/FPLA-3121